src/main/webapp/static/js/controllers/scheduling_controller.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * 
+ */
+ 
+ 'use strict';
+ 
+angular
+	.module('myApp')
+	.controller('SchedulingController', SchedulingController) 
+
+SchedulingController.$inject = ['SchedulingService','$log']
+
+function SchedulingController(SchedulingService, $log){
+	
+	var vm = this;
+	vm.job = {id: null, customerName: '', address: '', email: '',
+		phone: '', data: '', description: ''};
+	vm.jobs = [];
+	
+	vm.submit = submit;
+	vm.update = update;
+	vm.deleteJob = deleteJob
+	vm.reset = reset;
+	
+	getAllJobs();
+	
+	function getAllJobs() {
+		SchedulingService.getAllJobs()
+		.then(
+			function(e) {
```

Comment:
spell out the variable here, it looks like your normal return is an execption. This 'e'  should be something like "data"